Automattic/wp-calypso
# Onboarding import module: Reorganize base/common styles inside the package

#### Changes proposed in this Pull Request

* Reorganize base/common styles inside the package 
* Created `base.scss` file
* Created `component.scss` file

The current style is not touched. Everything remains the same as before the change, which contains file reorganization for better usage of styles.

#### Testing instructions

* Just visit and check does styles remain the same
* check screens -> `/start/importer/capture?siteSlug={SITE_SLUG}`
* check screens -> `/start/from/importing/wordpress?from=&to={SITE_SLUG}`
* check screens -> `/start/import-light/static`
  *  `/start/import-light/static/scanning`
  *  `/start/import-light/static/colors`
  *  `/start/import-light/static/colors`
  *  `/start/import-light/static/summary`

Related to #57131
